crates/store/re_protos/proto/rerun/v1alpha1/sdk_comms.proto

```diff
@@ -18,6 +19,8 @@ service MessageProxyService {
   //            It may allow us to amortize the overhead of the gRPC protocol.
   rpc WriteMessages(stream WriteMessagesRequest) returns (WriteMessagesResponse) {}
   rpc ReadMessages(ReadMessagesRequest) returns (stream ReadMessagesResponse) {}
+
+  rpc WriteTable(WriteTableRequest) returns (WriteTableResponse) {}
```

Comment:
```suggestion
  rpc WriteTable(WriteTableRequest) returns (WriteTableResponse) {}
  rpc ReadTables(ReadTablesRequest) returns (stream ReadTablesResponse) {}
```

(it's plural on purpose, you are writing one table, but reading potentially many)